Adicionado tela de inserção de novos produtos

duccl opened the pull request:
Foi usada uma hash para tal ação. Pois sua complexidade (para casos comuns) para inserção, remoção e busca são O(1)

duccl committed:
Adicionado tela de inserção de novos produtos

Foi usada uma hash para tal ação. Pois sua complexidade (para casos comuns) para inserção, remoção e busca são O(1)

duccl committed:
Add files via upload

duccl committed:
Add files via upload

duccl committed:
Add files via upload

duccl committed:
Delete mainmenu.cpp

duccl committed:
Delete mainmenu.h

duccl committed:
Delete mainmenu.ui

duccl committed:
Add files via upload

duccl committed:
Add files via upload

duccl committed:
Add files via upload

duccl committed:
Update QtGuiApplication1.h

duccl committed:
Add files via upload

duccl committed:
Add files via upload

duccl committed:
Add files via upload

duccl committed:
Add files via upload

duccl committed:
Merge branch 'master' into duccl-homologacao_v2

duccl commented:
Também foram adicionados arquivos para permitir o uso do projeto no Qt Creator. Isso foi feito pois o visual studio cria muitos arquivos que não são necessários a priori

gabrielfigcosta committed:
Tela de exclusão

Foi adicionada uma tela para excluir os produtos cadastrados. Usei o método pronto da hash para remoção para realizar isso

duccl committed:
Troca na passagem de parâmetros

Foi alterado para receber um ponteiro, ao invés da referência de um ponteiro para os forms filhos do menu principal

kelvinlins committed:
Tela de Consulta

Foi adicionada uma tela de consulta pelo nome que abre uma caixa de texto com o produto pesquisado.

kelvinlins committed:
Add files via upload

kelvinlins commented:
Faltou eu ter adicionado outros arquivos de dependência para o projeto buildar. Esse meu último commit foi para inclui-los.

kelvinlins committed:
Bug do build inifinto

kelvinlins committed:
Merge pull request #4 from duccl/tentando-resolver-bug-build

Bug do build inifinto foi resolvido quando apertei em "run qmake"

kelvinlins committed:
Revert "Bug do build inifinto"

kelvinlins committed:
Merge pull request #5 from duccl/revert-4-tentando-resolver-bug-build

Revert "Bug do build inifinto"

kelvinlins committed:
Delete trabalhoControleDeEstoque.pro.user.8976280

kelvinlins committed:
Delete trabalhoControleDeEstoque.pro.user.b3e8a96

kelvinlins committed:
Delete trabalhoControleDeEstoque.pro.user

kelvinlins committed:
Teste para tentar reveter o erro do qmake

kelvinlins commented:
Bug resolvido :)

duccl commented:
> Bug resolvido :)

Acho que ainda não deu certo, tentei aqui e continua dando o erro

duccl committed:
Novo upload para reverter bug

Tentei alterar o .pro
Parece que esse bug tem a ver com um timestamp no arquivo

Commit:
Repositorio repetido removido

duccl committed:
Correção bug na consulta de produtos

Quando realizávamos a consulta de um produto que não existia, o programa parava de funcionar. Isso foi causado por conta do escopo de retorno. Foi criada uma variável sentinela na Hash e na LDDE para resolver o problema.
